src/file-uploader/styled-components.js

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import {styled} from '../styles/index.js';
+import type {StylePropsT} from './types.js';
+
+export const StyledFileDragAndDrop = styled('div', (props: StylePropsT) => {
+  return {
+    alignItems: 'center',
+    backgroundColor: props.$theme.colors.fileUploaderBackgroundColor,
+    borderColor: props.$isDragActive
+      ? props.$theme.colors.fileUploaderBorderColorActive
+      : props.$theme.colors.fileUploaderBorderColorDefault,
+
+    borderStyle: 'dashed',
+    borderRadius: props.$theme.borders.radius200,
```

Comment:
Not sure if we need to have the `$theme.borders.useRoundedCorners` check here. Something to consider.